src/main/java/view/BreakSessionView.java

```diff
@@ -61,13 +71,17 @@ public BreakSessionView(BreakSessionViewModel breakSessionViewModel, BreakSessio
         logOutSettings = createButton("Log Out");
         startTimerButton = createButton("Start Timer");
 
+        // Set preferred size to match buttons
+        Dimension buttonSize = new Dimension(150, 40);
```

Comment:
Import some constants for the magic numbers. Dimension button should also be declared final.